### PR TITLE
Made the Heading Bold

### PR DESCRIPTION
**Pull request by Arcturus22**

### CODE REVIEW

**Comment by Arcturus22** [comment]
This Pull Request is regarding Issue #88

File: index.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
 
     <!-- Title -->
 
-    <title>CALCULATOR | YOSHITHA</title>
+    <title><b>CALCULATOR | YOSHITHA</b></title>
```
Comment (YoshithaRathnayake) [inline index.html:56]: This isn't the one wanted to bold its the main topic "CALCULATOR | YOSHITHA" @Arcturus22

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by Arcturus22** [comment]
Please verify @YoshithaRathnayake

**Review by YoshithaRathnayake — APPROVED** [review]
Perfect!
Thank You @Arcturus22

**Comment by YoshithaRathnayake** [comment]
@all-contributors please add @Arcturus22 for infrastructure, tests and code

**Comment by Arcturus22** [comment]
> @all-contributors please add @Arcturus22 for infrastructure, tests and code

Thanks a lot for this honour.😊

**Comment by Arcturus22** [comment]
Added a new Design to the README.md

**Comment by YoshithaRathnayake** [comment]
> > @all-contributors please add @Arcturus22 for infrastructure, tests and code
> 
> @YoshithaRathnayake Thanks a lot for this honour.😊

Pleasure 😊 @Arcturus22

**Comment by YoshithaRathnayake** [comment]
> Added a new Design to the README.md

Can you make a new issue and put a screenshot of the design @Arcturus22

**Comment by Arcturus22** [comment]
> > Added a new Design to the README.md
> 
> Can you make a new issue and put a screenshot of the design @Arcturus22 

No,I did not mean that.
Please ignore that message.

**Comment by YoshithaRathnayake** [comment]
> > > Added a new Design to the README.md
> > 
> > 
> > Can you make a new issue and put a screenshot of the design @Arcturus22
> 
> No,I did not mean that. Please ignore that message.

Ok I will ignore that @Arcturus22

**Pull request merged**